src/common/components/vanilla/notification/index.js

```diff
@@ -13,25 +13,14 @@ export default class Notification extends Component {
           { status && this.getStatus(this.props.status) }
           { message }
           { this.props.children }
-          { action && actionText &&
```

Comment:
Notification reused from what we have in MU.
But we don't use 'actions' anywhere here. Instead we have new icon X button with remove callback.